### ISSUE TITLE
allow setting a tree as `unmanaged`

### ISSUE THREAD

**Issue by igalic** [voxpupuli/modulesync#89]
Currently, in order to disable acceptance tests boilerplate, this is the configuration needed:
```yaml
spec/acceptance/nodesets/debian-78-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/ubuntu-server-1404-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/centos-66-x64-pe.yml:
  unmanaged: true
spec/acceptance/nodesets/debian-82-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/centos-511-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/centos-66-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/debian-8.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/debian-7.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/ubuntu-14.04.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/centos-6.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/ubuntu-16.04.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/centos-5.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/ubuntu-12.04.yml:
  unmanaged: true
spec/acceptance/nodesets/docker/centos-7.yml:
  unmanaged: true
spec/acceptance/nodesets/ubuntu-server-1604-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/ubuntu-server-1204-x64.yml:
  unmanaged: true
spec/acceptance/nodesets/centos-72-x64.yml:
  unmanaged: true
```

to my shame, a lot of modules don't have acceptance tests, and, yet, i'd still like to be able to use modulesync and @voxpupuli's configurations, because it's *really* convenient.

**Comment by igalic** [comment]
according to https://github.com/voxpupuli/modulesync/blob/master/features/update.feature#L173
this should already be supported.

**Comment by rnelson0** [comment]
Have you confirmed that it works in reality as well as the spec tests?

**Comment by igalic** [comment]
i believe to have confirmed in reality that it doesn't work.

**Comment by james-powis** [comment]
I don't have any answers at this time as to the issue described, however the test in question was massively flawed in its logic... I have a PR to update the test: https://github.com/voxpupuli/modulesync/pull/95

**Comment by james-powis** [comment]
Directory unmanaged was done on Dec 15, 2015, in this commit: https://github.com/voxpupuli/modulesync/commit/98aac8cca6a463b33e1d49836bc3315e4eed092a

Where as the latest tag 0.6.1 was cut on Aug 13, 2015 as such unless your gem file is pulling from the master branch of the modulesync git repo, this feature will be unavailable. 

Lets chalk this up to: https://github.com/voxpupuli/modulesync/issues/86 and https://github.com/voxpupuli/modulesync/issues/75

Special note... I have seen some oddities in my limited exposure to pulling form this projects master branch, consider it a "there be dragons" warning...

**Comment by rnelson0** [comment]
@james-powis Can you open issues for any dragons you've found?

@danzilio got some time this month to cut a new release? Anything I can do to help?

**Comment by james-powis** [comment]
@rnelson0 and @igalic 
Here is the issue I was encountering with master branch HEAD:
https://github.com/voxpupuli/modulesync/issues/92

However, I did confirm directory exclude does work in master branch:
```
19:52:04:~/git/puppet/modulesync_configs
jrp@MacBook-Pro (master *%=)> bundle exec msync update -f example_module 
Syncing example_module
Cloning repository fresh
Cloning from git@gitlab.viawest.com:puppet/example_module.git
Creating new branch modulesync
Not managing foo/1 in example_module
Not managing foo/2 in example_module
Not managing foo/3 in example_module
Not managing foo/4 in example_module
Not managing foo/5 in example_module
Not managing foo/6 in example_module
Not managing foo/7 in example_module
Not managing foo in example_module
git '--git-dir=/Users/jrp/git/puppet/modulesync_configs/modules/example_module/.git' '--work-tree=/Users/jrp/git/puppet/modulesync_configs/modules/example_module' commit '--message=Update from viawest modulesync_config'  2>&1:On branch modulesync
Changes not staged for commit:
	modified:   CONTRIBUTING.md

no changes added to commit
19:52:11:~/git/puppet/modulesync_configs
jrp@MacBook-Pro (master *%=)> cat modules/example_module/.sync.yml 
---
.gitlab-ci.yml:
  acceptance:
    - name: centos-6
    - name: centos-7
foo:
  unmanaged: true
```

**Comment by domcleal** [comment]
Thanks for checking @james-powis, it's just that there hasn't been a release. #92/#99 should fix the bug you found, and #75 will track getting this released.
